src/components/input/input.jsx

```diff
@@ -67,6 +67,8 @@ Input.propTypes = {
   autoFocus: PropTypes.bool,
   className: PropTypes.string,
   cssContainer: PropTypes.string,
+  controlRef: PropTypes.object, // eslint-disable-line react/forbid-prop-types
```

Comment:
I think this is a good use case to pass refs as props since we want to pass two different refs through, but let me know if we have a better pattern for this.